fluent-package/msi/build.bat

```diff
@@ -1,4 +1,5 @@
 SET SRC_DIR=%~dp0
+SET MSYS2_PATH="C:\tools\msys64"

```

Comment:
Enable chocolatey's MSYS2 while building.

Reply:
https://github.com/fluent/fluent-package-builder/actions/runs/7928743530/job/21647591198?pr=620

`ridk install 3` succeeds on building Docker image, but `ridk install 1 3` fails on `rake msi:selfbuild` in the Docker image.
I'm not sure but it seems that later one (especially the argument '1') isn't needed since MSYS2 is already installed.

Reply:
Above comment is completely wrong.

`MSYS2_PATH` is available as of RubyInstaller 3.2.3 while this revision installs 3.2.2.

Reply:
I've changed approach.
Skipping external MSYS2 isn't good approach because MSYS2 isn't installed under c:\opt\fluent even if installed by `C:\opt\fluent\bin\ridk install 1`.
Instead, just ignore `Encoding::UndefinedConversionError`.